source/quickstart.txt

```diff
@@ -219,6 +220,34 @@ The API documentation provides details on using the following endpoints:
    * - :ref:`c2c-api-reverse`
      - Reverses the direction of a committed sync operation.
 
+One-Time Sync
+-------------
+
+For a one-time sync, after initializing data synchronization, use the
+:ref:`progress <c2c-api-progress>` command to see the status of the
```

Comment:
[nit] Progress is an endpoint, not a command, so we should reword this sentence to avoid confusing users. Typically, we refer endpoints by "calling" them (i.e. "after initializing data synchronization, **call** the :ref:`progress <c2c-api-progress>` endpoint...")